code/datums/components/ai/controller.dm

```diff
@@ -0,0 +1,42 @@
+/datum/component/controller
```

Comment:
Controller for what?

This name is awful.

Reply:
This is what it was called by N3X15 and I just ported it over, won't address with this PR